modules/cms/src/admin/schema.admin.ts

```diff
@@ -71,31 +80,17 @@ export class SchemaAdmin {
       throw new GrpcError(status.INTERNAL, e.message);
     });
 
-    return { schemas, count };
-  }
-
-  async getSchemasFromOtherModules(
-    call: ParsedRouterRequest
-  ): Promise<UnparsedRouterResponse> {
-    const allSchemas = await this.database.getSchemas().catch((e: Error) => {
-      throw new GrpcError(status.INTERNAL, e.message);
-    });
-
-    const schemasFromCMS = await _DeclaredSchema.getInstance().findMany({
-      ownerModule: 'cms', name: { $nin: CMS_SYSTEM_SCHEMAS }
-    });
-    const schemaNamesFromCMS = (schemasFromCMS as _DeclaredSchema[]).map(
-      (schema: _DeclaredSchema) => schema.name
-    );
-    const schemasFromOtherModules = allSchemas.filter((schema: any) => {
-      return !schemaNamesFromCMS.includes(schema.name);
-    });
+    for (const schema of schemas) {
+      if (schema.ownerModule !== 'cms') { // || CMS_SYSTEM_SCHEMAS.includes(schema.name)) {
+        schema.modelOptions.conduit.cms = {
+          enabled: true,
+          crudOperations: false,
+          authentication: false, // here due to type restriction
```

Comment:
This doesn't look particularly clean imo. Maybe Conduit-UI should contain this logic instead?

Reply:
why would this be added to every schema? These values make no sense when the schema isn't owned by the CMS. They won't be able to alter them so why include them?

Reply:
Technically it shouldn't. Conduit-UI assumes it does, but yeah, I agree it shouldn't be here.
I'll remove this and ping for review/merge after confirming Conduit-UI works as expected.